go/api/commands.go

```diff
@@ -782,4 +782,21 @@ type ConnectionManagementCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/ping/
 	PingWithMessage(message string) (string, error)
+
+	// Echo the provided message back.
+	// Parameters:
+	//  string message
```

Comment:
Please copy docs from another clients

Reply:
fix in commit 2ac4523fe5714d35e32304c4d6f7cf4e9ae74067